pylab/accounts/forms.py

```diff
@@ -13,11 +13,15 @@ class UserProfileForm(forms.ModelForm):
 
     class Meta:
         model = accounts_models.UserProfile
-        fields = ('first_name', 'last_name', 'email', 'language')
+        fields = ('first_name', 'last_name', 'email', 'language', 'accepted_terms')
         help_texts = {
             'email': _(
                 "Will be used for communication. If you want not to get any emails, leave this field empty."
             ),
+            'accepted_terms': _(
+                "[Terms of serivce](http://pylab.lt/terms/). Basically, you accept that all your "
```

Comment:
Consider using `[Terms of serivce](http://pylab.lt/terms/){:target=_blank}` to fore link to TOS open in new tab instead of replacing current page, possibly loosing filled form values.